src/app/services/search/search.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class SearchService {
+
+  apiUrl = "https://landville-backend-web-api.herokuapp.com/api/v1/properties/?";
```

Comment:
I noticed we have a custom `HttpService` setup in `/services/http.service.ts`. I think using it would be better since it includes this URL in the request so you wouldn't have to pass it in manually here... unless there was a specific reason you chose not to use it.